.circleci/verbatim-sources/job-specs/binary-job-specs.yml

```diff
@@ -326,14 +326,6 @@
 
   binary_windows_upload:
     <<: *binary_windows_params
-    parameters:
```

Comment:
Is there a way to specify, that we want to run those jobs on windows-medium rather than windows-xlarge machine?